### ISSUE TITLE
confusing file.replaced.warning.previous.warningMessage text 

### ISSUE THREAD

**Issue by qqmyers** [IQSS/dataverse#7367]
The bundle message says "You can not edit a file that has been replaced in a previous dataset version. In order to edit it you must go to the most recently published version of the file." - 'previous' should be 'subsequent' or similar? Also, since one can delete a file from a draft version (and with #7337 also replace in draft), should this also change to 'most recent version'?

**Comment by jggautier** [comment]
I agree "previous" doesn't make sense. I just tested this on Demo Dataverse by:
- creating a new version by using the "File Replace" feature to replace a file with another file
- publishing that new version
- going back to a previous dataset version to try to edit the metadata of the file that was replaced
- and getting a warning message with that text

<img width="610" alt="Screen Shot 2020-10-28 at 11 06 07 AM" src="https://user-images.githubusercontent.com/18374574/97454953-b2cd4b80-190d-11eb-8d5c-f52274eaac17.png">

(I'm not sure how the wording should change to accommodate the PR for enabling the "file replace" feature in draft datasets. Not familiar with how that PR will work. Just thought I'd add steps to reproduce and a screenshot for anyone following along :)
